go/flexibletable/table.go

```diff
@@ -42,6 +43,71 @@ func (t *Table) Insert(row Row) error {
 	return nil
 }
 
+func (t *Table) breakOnLineBreaks() error {
+
+	// so that there's no need to resize if there's no line break
+	broken := make([]Row, 0, len(t.rows))
+
+	for _, row := range t.rows {
+
+		for notEmpty := true; notEmpty; {
```

Comment:
How about this little change? If it does the same thing, I think it's a little clearer.
```
		notEmpty := true
		for notEmpty {
			newRow := make(Row, 0, len(row))
			notEmpty = false
```